Headers/AppLovinSDK.framework/Headers/ALSdk.h

```diff
@@ -0,0 +1,226 @@
+//
+//  ALSdk.h
+//  AppLovinSDK
+//
+//  Created by Basil Shikin on 2/1/12.
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "ALSdkSettings.h"
+#import "ALAdService.h"
+#import "ALEventService.h"
+#import "ALVariableService.h"
+#import "ALUserService.h"
+#import "ALSdkConfiguration.h"
+#import "ALErrorCodes.h"
+#import "ALMediationProvider.h"
+#import "ALUserSegment.h"
+#import "MAMediatedNetworkInfo.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This is a base class for the AppLovin iOS SDK.
+ */
+@interface ALSdk : NSObject
+
+
+#pragma mark - High Level SDK Properties
+
+/**
+ * The current version of the SDK. The value is in the format of “<var>Major</var>.<var>Minor</var>.<var>Revision</var>”.
+ */
+@property (class, nonatomic, copy, readonly) NSString *version;
+
+/**
+ * The current version of the SDK in numeric format.
+ */
+@property (class, nonatomic, assign, readonly) NSUInteger versionCode;
+
+/**
+ * This SDK’s SDK key.
+ */
+@property (nonatomic, copy, readonly) NSString *sdkKey;
+
+/**
+ * This SDK’s SDK settings.
+ */
+@property (nonatomic, strong, readonly) ALSdkSettings *settings;
+
+/**
+ * The SDK configuration object provided upon initialization.
+ */
+ // [PLP] who provided it to whom? Is this "the SDK configuration object you provided when you initialized the SDK"? If so, how do you provide it?
```

Comment:
When the SDK initializes it basically just fires off an endpoint to ourservers, which returns back this configuration object. The way it's provided is implicit, since it's a property of our main SDK object that pubs can easily access throughout their integrations.